CONTRIBUTING.md

```diff
@@ -26,6 +26,12 @@ Please follow these steps to report such a problem
    TEST_URL=https://developer.github.com/v3/repos/#get ./node_modules/.bin/tap test/integration/single-endpoint-test.js
    ```
 
+   and for GitHub Enterprise:
+
+   ```
+   GHE_VERSION=2.17 TEST_URL=https://developer.github.com/enterprise/2.17/v3/repos/#get ./node_modules/.bin/tap test/integration/single-endpoint-test.js
+   ```
```

Comment:
Ideally we make this work by just passing `TEST_URL` as it also includes the GHE version, but that’s not urgent, let’s get back to it another day 👍

Reply:
will track this via #446